pyani_plus/private_cli.py

```diff
@@ -440,8 +440,9 @@ def log_anim(
     # after the computation has finished (on the same machine)
     nucmer_tool = tools.get_nucmer()
 
-    r_aligned_bases, q_aligned_bases, identity, sim_errors = method_anim.parse_delta(
-        deltafilter
+    # Note inconsistency in how nucmer uses query vs reference and flips order
+    query_aligned_bases, subject_aligned_bases, identity, sim_errors = (
+        method_anim.parse_delta(deltafilter)
     )
```

Comment:
I don't think this change is needed. `method_anim.parse_delta` reports values in the following order:

```python
    return (
        get_aligned_bases_count(regions_ref),
        get_aligned_bases_count(regions_qry),
        avrg_identity,
        sim_error,
    )
```

Reply:
What I think needs to be changed is line 449, as I think this is flipped
```
used_query, used_subject = deltafilter.stem.split("_vs_")
```

Reply:
For context... `rule_delta` runs the following command:
```python
{params.nucmer} -p {wildcards.outdir}/{wildcards.genomeA}_vs_{wildcards.genomeB} --mum {input.genomeA} {input.genomeB}
```
According to nucmer documentation, they refer to them as reference (genomeA) and query (genomeB)

```bash
USAGE: nucmer  [options]  <Reference>  <Query>
```

`rule_delta` uses the same format eg. genomeA_vs_genomeB:

```python
{outdir}/{genomeA}_vs_{genomeB}.filter
```

Reply:
Ah! I understood we had the convention in pyANI plus (and legacy pyANI), that the intermediate files were all called ``<query>_vs_<subject>.<extension>`` and strongly feel that should be used consistently.

It seems then that we should swap this:

``{params.nucmer} -p {wildcards.outdir}/{wildcards.genomeA}_vs_{wildcards.genomeB} --mum {input.genomeA} {input.genomeB}``

and:

```python
query_aligned_bases, subject_aligned_bases, identity, sim_errors = (
    method_anim.parse_delta(deltafilter)
)
```

to be this:

``{params.nucmer} -p {wildcards.outdir}/{wildcards.genomeA}_vs_{wildcards.genomeB} --mum {input.genomeB} {input.genomeA}``

and:

```python
subject_aligned_bases, query_aligned_bases, identity, sim_errors = (
    method_anim.parse_delta(deltafilter)
)
```

Then it should all be consistent?

Reply:
If we go this rount we will need to change this for `dnadiff` method too.